### PR TITLE
Nodejs week3

### PR DESCRIPTION
**Pull request by Aditi-Sawardekar**

### CODE REVIEW

File: nodejs/week3/api/contacts.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+// http://localhost:3000/api/contacts/
+// http://localhost:3000/api/contacts/?sort=first_name%20ASC
+// http://localhost:3000/api/contacts/?sort=last_name%20DESC
+
+contactsAPIRouter.get("/", async (req, res) => {
+    let query = database.select("*").from("contacts");
+   
+    if ("sort" in req.query) {
+        const validTags = ["first_name ASC", "first_name DESC", "last_name ASC", "last_name DESC"]
```
Comment (itsgauravsaxena) [inline nodejs/week3/api/contacts.js]: This should be case insenstive - means small case asc and desc should also work.

File: nodejs/week3/api/contacts.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import express from 'express';
+import database from "../database.js";
+
+const router = express.Router();
+const apiRouter = express.Router();
+const contactsAPIRouter = express.Router();
+
+// http://localhost:3000/api/contacts/
```
Comment (itsgauravsaxena) [inline nodejs/week3/api/contacts.js]: Remove commented code - always a good practise

**Review by itsgauravsaxena — APPROVED** [review]
MIssing recording or screenshot for sql injection issue Else looks good :)

File: nodejs/week3/api/contacts.js (hunk 2 above)
Comment (Aditi-Sawardekar) [reply nodejs/week3/api/contacts.js]: Removed commented code.

**Comment by Aditi-Sawardekar** [comment]
Hi Gaurav, 
Thanks for reviewing my code. I have recorded the Video to Demonstrate SQL Injection. I will upload it soon.

File: nodejs/week3/api/contacts.js (hunk 1 above)
Comment (Aditi-Sawardekar) [reply nodejs/week3/api/contacts.js]: Made sort case sensitive.

**Comment by Aditi-Sawardekar** [comment]
https://user-images.githubusercontent.com/107069781/221312643-eb08a3d8-e07f-4abf-a29c-0cd171f2888a.mp4

File to demonstrate SQL Injection.

**Comment by Aditi-Sawardekar** [comment]
Hi Gaurav,
Thanks for reviewing my code and approving the merge request.

**Pull request merged**